libvirt-python.rb

```diff
@@ -0,0 +1,26 @@
+class LibvirtPython < Formula
+  desc "Python bindings for libvirt"
+  homepage "http://www.libvirt.org"
+
+  stable do
+    url "http://libvirt.org/sources/python/libvirt-python-1.3.1.tar.gz"
+    sha256 "7143b922a9bd66a5e199b13316fa4266cbffc23db01c99bce37216a1eb9118f7"
+  end
```

Comment:
addressed in https://github.com/vicamo/homebrew/commit/4dc43beeb24003c497cc271dd8dc35d7b473511e